__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -263,7 +263,7 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			);
 			const cellSize = await cell.getSize();
 			const cellLocation = await cell.getLocation();
-			const scrollOffset = isAndroid() ? 350 : 100;
+			const scrollOffset = isAndroid() ? 400 : 150;
```

Comment:
I'm wondering what changed so we need to update this value 🤔 .

Reply:
Ah, I've just noticed that now that the Columns block has the Text/Background setting.

Reply:
Sorry! I forgot to put this in the PR description, it cherry picks changes from https://github.com/wordpress-mobile/gutenberg-mobile/pull/5683 since now the Color options are showing we needed to update the offset.